Documentation/templates/strings/objc.md

```diff
@@ -0,0 +1,53 @@
+## Template Information
+
+| Name      | Description       |
+| --------- | ----------------- |
+| File names | strings/objc-string-h.stencil & strings/objc-string-m.stencil |
+| Configuration example | <pre>strings:<br />  inputs: path/to/Localizable.strings<br />  outputs:<br />    - templateName: objc-string-h<br />      output: Localizable.h<br />    - templateName: objc-string-m<br />      output: Localizable.m</pre> |
+| Language | Objective-C |
+| Author | Eric Slosser |
+
+## When to use it
+
+- When you need to generate *Objective-C* code.  Perhaps because you have a pure Objective-C project and don't want to add a Swift file to it (thus triggering the inclusion of the Swift runtime into your build).  Or perhaps because you can't find a template that generates Swift that can be accessed from Objective-C.
+
+## Customization
+
+You can customize some elements of this template by overriding the following parameters when invoking `swiftgen`. See the [dedicated documentation](../../ConfigFile.md).
+
+| Parameter Name | Default Value   | Description                                                  |
+| -------------- | --------------- | ------------------------------------------------------------ |
+| `noComments`   | N/A             | Setting this parameter will disable the comments describing the translation of a key. |
+| `headerName`   | "Localizable.h" | The name of the ObjC header generated by the `objc-string-h` template.  The generated `.m` file will `#include` "{{headerName}}".  This parameter should have the same value as the `output` file name used for the `template: objc-string-h` entry in your config file. |
```

Comment:
The config example seems wrong. Should be something like:
```yaml
templateName: objc-h
```

Reply:
good catch, thanks